storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst

```diff
@@ -85,3 +94,54 @@ Command-line options
 .. option:: --insecure 
 
    Do not verify servers certificate
+
+.. _swift_auth:
+
+Swift authentication options
+----------------------------
+
+Swift specification describe several `authentication options <http://docs.openstack.org/developer/swift/overview_auth.html>`_. |xbcloud| can authenticate against keystone with API version 2 and 3. Specifies authentication version (1.0 for TempAuth, 2.0 for Keystove v2.0, 3 for Keystone v3) can be specified with :option:`--swift-auth-version`. Default is 1.0
```

Comment:
Sentense "Specifies authentication version (1.0 for TempAuth, 2.0 for Keystove v2.0, 3 for Keystone v3) can be specified with :option:`--swift-auth-version`. Default is 1.0" is probably left over.